backend/protocol_rpc/server.py

```diff
@@ -75,6 +75,7 @@ def run_socketio():
         host="0.0.0.0",
         allow_unsafe_werkzeug=True,
     )
+    logging.getLogger("werkzeug").setLevel(logging.ERROR)
```

Comment:
can we set this only if there's an envvar like "DEBUG" or "DEV"? These logs might have important warnings when running in production

Reply:
Done!